.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "modules/Ben.Demystifier"]
 	path = modules/Ben.Demystifier
 	url = ../Ben.Demystifier.git
+	ignore = dirty
```

Comment:
if you're making changes to it, how does it work?
It doens't show that you've got changes in it?

I believe the biggest question is: Why do you have local changes on the submodule?
```suggestion
	ignore = dirty

```

Reply:
changes are ignored, it could be that your submodule is outdated or updated compared to the main branch.

Reply:
Not sure that's what we want though